user/api.go

```diff
@@ -232,6 +232,11 @@ func (a *Api) CreateUser(res http.ResponseWriter, req *http.Request) {
 			a.logMetricForUser(newUser.Id, "usercreated", sessionToken.ID, map[string]string{"server": "false"})
 			res.Header().Set(TP_SESSION_TOKEN, sessionToken.ID)
 			a.sendUserWithStatus(res, newUser, http.StatusCreated, false)
+			if err := a.userEventsNotifier.NotifyUserCreated(req.Context(), *newUser); err != nil {
```

Comment:
`a.sendUserWithStatus(res, newUser, http.StatusCreated, false)` must be placed after this block

Reply:
I suggest removing this event completely if it's not used by marketo.

Reply:
ok wasnt sure if I should add the notify to all create and update events. removing